### PR TITLE
Chore: Move to node 18 again

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- move to node 18

**Why do we need this feature?**

- node 18 is in active LTS as of October 25th

**Who is this feature for?**

- everyone

**Which issue(s) does this PR fix?**:

Follow up to https://github.com/grafana/grafana/pull/58570

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-ui/src/slate-plugins/braces.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { Annotation } from 'slate';
 import { Plugin } from 'slate-react';
 import { v4 as uuidv4 } from 'uuid';
 
-const BRACES: any = {
+const BRACES: Record<string, string> = {
```
Comment (ashharrison90) [inline packages/grafana-ui/src/slate-plugins/braces.ts:5]: this is to force us to build a canary version when this is merged to check everything has worked ok 👌

**Review by kaydelaney — APPROVED** [review]
🎉🎉🎉

**Review by jackw — APPROVED** [review]
LGTM! 🚀 👏

**Review by chri2547 — APPROVED** [review]
lgtm!

**Pull request merged**